rust_src/src/javascript.rs

```diff
@@ -1889,6 +1898,137 @@ pub fn js_tick_event_loop(handler: LispObject) -> LispObject {
     lisp::remacs_sys::Qnil
 }
 
+// We overwrite certain subcommands to allow interfacing with emacs-lisp
+// other subcommands, we will use deno's default implementation
```

Comment:
> // We overwrite certain subcommands to allow interfacing with emacs-lisp
// other subcommands

I don't quite understand this comment. Is there an "and" missing ?

Reply:
Updated the wording for this.